NIPostsReload/Presentation/Scenes/NIPostFeedModule/Presenter/NIPostFeedPresenter.swift

```diff
@@ -50,6 +53,19 @@ final class DefaultNIPostFeedPresenter: NIPostFeedPresenter {
         postViewState.posts.count
     }
     
+    func getSelectedFeedType() -> PostFeedType {
+        selectedFeedType
+    }
+    
+    func updateSelectedFeedType(with type: PostFeedType) {
```

Comment:
this is not a good name for the presenter method, as if the controller has already decided what to do instead of the presenter